pkg/serverless/appsec/appsec.go

```diff
@@ -70,7 +71,11 @@ func newAppSec() (*AppSec, error) {
 		return nil, err
 	}
 
-	handle, err := waf.NewHandle([]byte(appsec.StaticRecommendedRules), cfg.obfuscator.KeyRegex, cfg.obfuscator.ValueRegex)
+	var rules map[string]any
+	if err := json.Unmarshal([]byte(appsec.StaticRecommendedRules), &rules); err != nil {
```

Comment:
I'm a bit worried about that. If I understand correctly this will unmarshal 7k line of JSON (https://github.com/DataDog/appsec-internal-go/blob/main/appsec/rules.json)

Before we did not need to unmarshal it?

Reply:
It was unmarshaled by `waf.NewHandle` but we changed its API to now accept a Go value directly. So there is nothing new here: this is done once and at init time, only when appsec is enabled.

Reply:
thanks for the clarification!